pkg/systemd/init.js

```diff
@@ -148,6 +148,14 @@ $(function() {
     var units_by_path = { };
     var path_by_id = { };
 
+    var type_selectors = {
+        "\\.target$": _("All Targets"),
+        "\\.service$": _("All Services"),
+        "\\.socket$": _("All Sockets"),
+        "\\.timer$": _("All Timers"),
+        "\\.path$": _("All Paths")
+    };
```

Comment:
I don't quite understand this: we already have the tab headers for "Targets", "Services", "Sockets", and so on. Why do we need another selector for the type?

Also, this seems to have nothing to do with "filtering by autostart". Separate commit?

Reply:
Yeeah, I was expecting this will receive a comment. (I am not happy with it but cannot think of better way of doing this).
This is here, because when you change type for example from services to timers you want the dropdown contain "All Timers" and not "All Services" etc.
Was thinking how to do this so it still can be translated, can be dynamic and also I need to check if the current selection is not one of "All ..." and update that as well as  first item in the dropdown.

Reply:
Thanks, this makes more sense now. So conceptually this is "All enablement states", but that sounds too ugly. Perhaps "Any Startup", to relate it to the column header? But I'll leave this between @garrett and you, I just wanted to understand what's going on.

Reply:
It was suggested by @andreasn, so maybe he can join this as well

Reply:
If you're concerned about repetition, _All_ could work as a label.

Reply:
Or _Everything_

Reply:
Changed to `All`